pennylane/templates/embeddings/iqp.py

```diff
@@ -132,7 +132,7 @@ def circuit(features):
 
         .. code-block:: python
 
-            from pennylane import numpy as np
+            import numpy as np
```

Comment:
I'm not sure about changing any of the docstrings.

Reply:
every docstring change was an accident 😄 I'll double-check for others that I might have accidentally changed and undo them